fix: update the traffic inspector section of airship (#1458)
ngrok/ngrok-docs: docs/integrations/airship/webhooks.mdx

### Inspecting requests

> ngrok's [Traffic Inspector](https://dashboard.ngrok.com/traffic-inspector) captures all requests made through your ngrok endpoint to your localhost app. Click on any request to view detailed information about both the request and response.

Reviewer: Style Guide preference: one sentence per line

```suggestion
ngrok's [Traffic Inspector](https://dashboard.ngrok.com/traffic-inspector) captures all requests made through your ngrok endpoint to your localhost app. 
Click on any request to view detailed information about both the request and response.
```

> **Note**: By default, accounts only collect traffic metadata to avoid exposing secrets. Enable full capture in your account Settings to capture complete request and response data.

Reviewer: I usually like to wrap this kind of thing in a callout instead of **Note**-ing it. Also nice to give the reader a link to their settings and some idea of where to look in there.

```suggestion
:::info
By default, accounts only collect traffic metadata to avoid exposing secrets. 
You must enable full capture in the Observability section of [your account settings](https://dashboard.ngrok.com/settings) to capture complete request and response data.
:::
```

> Use the Traffic Inspector to:

Reviewer: Style Guide currently dictates that ngrokisms should only be capitalized on first mention (I'm not sure I agree with this fwiw)

```suggestion
Use the traffic inspector to:
```

- Validate webhook payloads and response data
- Debug request headers, methods, and status codes
- Troubleshoot integration issues without adding logging to your app

### Replaying requests

Test your webhook handling code without triggering new events from Airship using the Traffic Inspector's replay feature:

> 1. Enable full capture in your ngrok account settings to capture complete request data.

Reviewer: this line feels redundant since we're already calling out that you need to do this earlier in the section. Maybe it's not necessary to call out earlier, and it would make more sense to move that info here?

1. Send a test webhook from Airship to generate traffic in your Traffic Inspector.

> 1. Select the request you want to replay in the [Traffic Inspector](https://dashboard.ngrok.com/traffic-inspector).

Reviewer: Don't think it's necessary to link out again after it's already been linked and mentioned a few times - ideally the reader knows what this is by this point! 

```suggestion
1. Select the request you want to replay in the traffic inspector.
```
